### ISSUE TITLE
is coc-eslint still maintained?

### ISSUE THREAD

**Issue by laoshaw** [neoclide/coc-eslint#146]
I could never get coc-eslint to work with coc.nvim with typescript, i.e. to give linter warns and errors inside vim while I'm writing the code, I noticed this project has not been updated for two years, and warns against newest typescript, is this still a good option to use with vim and typescript today?

**Comment by fannheyward** [comment]
I'm using coc/coc-eslint with TS 5.3 without any problems, any error messages?

**Comment by fannheyward** [comment]
> message says it has to be ts5.4.0 but a default install is now 5.4.2.

Any screenshot? I want to know who output this message, eslint or prettier or tsserver.

`npm create vue@latest` to create `vue-test` project, select to enable TypeScript, ESLint, Prettier

<img width="713" alt="image" src="https://github.com/neoclide/coc-eslint/assets/345274/9690b114-a577-4f94-8e66-9074c1f7dac0">

`npm list`

```
vue-test@0.0.0 /Users/fannheyward/src/vue-test
├── @rushstack/eslint-patch@1.7.2
├── @tsconfig/node20@20.1.2
├── @types/node@20.11.26
├── @vitejs/plugin-vue@5.0.4
├── @vue/eslint-config-prettier@8.0.0
├── @vue/eslint-config-typescript@12.0.0
├── @vue/tsconfig@0.5.1
├── eslint-plugin-vue@9.23.0
├── eslint@8.57.0
├── npm-run-all2@6.1.2
├── prettier@3.2.5
├── typescript@5.4.2
├── vite@5.1.6
├── vue-tsc@2.0.6
└── vue@3.4.21
```

extensions:

- coc-tsserver 2.1.4
- coc-eslint 1.6.0
- coc-prettier 9.3.2
- coc-volar 0.35.0

Tests:

`vim src/App.vue`, make some errors in script

<img width="842" alt="image" src="https://github.com/neoclide/coc-eslint/assets/345274/3c68d34b-3bf6-4ea4-a67f-cc2f9995773b">

Formatting works too.

> when editing standalone ts files, eslint and prettier does nothing

I don't know what you mean _standalone_ files, are they in TS project?

**Comment by chemzqm** [comment]
Yes, coc-eslint@3.0.13 have the same features of vscode-eslint@3.0.13
